include/inexor/vulkan-renderer/world/cube.hpp

```diff
@@ -105,6 +146,11 @@ class Cube : public std::enable_shared_from_this<Cube> {
     /// Indent a specific edge by steps.
     /// @param positive_direction Indent in  positive axis direction.
     void indent(std::uint8_t edge_id, bool positive_direction, std::uint8_t steps);
+    /// Rotate a cube on one axis.
+    /// @param axis Axis to rotate on (all values except must be 0 or a multiple of 4 [which indicates no rotation]).
+    ///             0 = rotate by 0°, 1 = rotate by 90°, 2 = rotate by 180°, 3 = rotate by 270°
+    ///             -1 = rotate by 270°, -2 = rotate by 180°, -3 = rotate by 90°
+    void rotate(const glm::vec<3, int8_t> &axis);
```

Comment:
Since we have only 3 rotation axes, we could replace this `glm::vec3` with something like `ROTATION_AXIS_X`...

Reply:
Maybe add this comment to the discussion about that method above. Easier to catch it all.